psclient/src/main/java/com/majeur/psclient/ui/BattleFragment.kt

```diff
@@ -71,6 +75,10 @@ class BattleFragment : BaseFragment(), BattleRoomMessageObserver.UiCallbacks, Vi
 
     val battleRunning get() = observer.battleRunning
 
+    fun battleType(): BattleRoomMessageObserver.BattleType {
+        return battleType
+    }
```

Comment:
You can already access battleType field value, this is redundent.
```suggestion
```

Reply:
Done!